### PR TITLE
ChelakhovLeonid/HW3

### PR DESCRIPTION
**Pull request by Chelakhovl**

### CODE REVIEW

File: Chelakhovl/HW3/1.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+zen_python = """1.Beautiful is better than ugly.
```
Comment (kolyasalubov) [inline Chelakhovl/HW3/1.py:1]: good job

File: Chelakhovl/HW3/2.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#вирішив задачу за допомогою зрізів
```
Comment (kolyasalubov) [inline Chelakhovl/HW3/2.py:1]: good idea

File: Chelakhovl/HW3/3.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#створюємо дві зміни в яких значення буде число
+
```
Comment (kolyasalubov) [inline Chelakhovl/HW3/3.py:2]: good idea

but look at this version of code
a, b = b, a

**Review by kolyasalubov — APPROVED** [review]

File: Chelakhovl/HW3/1.py (hunk 1 above)
Comment (Chelakhovl) [reply Chelakhovl/HW3/1.py:1]: > good job

thanks for you reaction

File: Chelakhovl/HW3/2.py (hunk 2 above)
Comment (Chelakhovl) [reply Chelakhovl/HW3/2.py:1]: > good idea

thanks for you comment

File: Chelakhovl/HW3/3.py (hunk 3 above)
Comment (Chelakhovl) [reply Chelakhovl/HW3/3.py:2]: > good idea
> 
> but look at this version of code a, b = b, a

thanks, Teacher=)
I changed file with You recomendation.

**Pull request merged**